build/common.props

```diff
@@ -2,12 +2,12 @@
   <PropertyGroup>
     <!-- Extensions can have independent versions and only increment when released -->
     <Version>3.0.0$(VersionSuffix)</Version>
-    <ExtensionsVersion>4.0.2$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
-    <CosmosDBVersion>3.0.8$(VersionSuffix)</CosmosDBVersion>
-    <HttpVersion>3.0.12$(VersionSuffix)</HttpVersion>
+    <ExtensionsVersion>4.0.3$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
+    <CosmosDBVersion>3.0.9$(VersionSuffix)</CosmosDBVersion>
+    <HttpVersion>3.0.13$(VersionSuffix)</HttpVersion>
```

Comment:
This was just incremented already yesterday in https://github.com/Azure/azure-webjobs-sdk-extensions/commit/26ad6c71344d794a8a1f79e27899e101d6df2160. For each of these bumps, can you verify that we need to bump them?

Reply:
Will update this repo with the release notes flow we have in the host. That will also help track this.